.github/workflows/format.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3.9
```

Comment:
Does python version makes a difference here?

Reply:
I don't think so, but we are updating to 3.9 on CIs.